pkgs/development/tools/misc/strace/default.nix

```diff
@@ -14,7 +14,7 @@ stdenv.mkDerivation rec {
 
   buildInputs = stdenv.lib.optional libunwind.supportsHost libunwind; # support -k
 
-  configureFlags = stdenv.lib.optional (stdenv.hostPlatform.isAarch64 || stdenv.hostPlatform.isRiscV) "--enable-mpers=check";
+  configureFlags = stdenv.lib.optional (!stdenv.hostPlatform.isx86_64 && !stdenv.hostPlatform.isi686) "--enable-mpers=check";
```

Comment:
Looks good!. But you can actually do ```stdenv.hostPlatform.isx86``` I believe (which covers both of these.)

Reply:
Yeah I just noticed :D